Libraries/Components/TextInput/TextInput.js

```diff
@@ -333,6 +334,31 @@ type IOSProps = $ReadOnly<{|
   textContentType?: ?TextContentType,
 |}>;
 
+// [TODO(macOS GH#774)
+export type SubmitKeyEvent = $ReadOnly<{|
+  key: string,
```

Comment:
This is called `code` on Windows - but it is a `key` on macOS

Should we unify it or not?
@Saadnajmi / @rozele

Reply:
Good catch. My first thought is we should align to here if it has an opinion: https://github.com/microsoft/react-native-windows/blob/main/vnext/proposals/active/keyboard-reconcile-desktop.md

I know that Windows and macOS differ a but about the keys. Windows differentiates "A, "keyA", and "GamepadA" I think?

Reply:
GamePadA example.. https://github.com/microsoft/react-native-windows/blob/f96ab5ed8ea8f3a51a7d2615ae339bace4b2fd1e/vnext/src/Libraries/Pressability/Pressability.windows.js#L595

Reply:
"KeyW" example https://github.com/microsoft/react-native-windows/blob/f96ab5ed8ea8f3a51a7d2615ae339bace4b2fd1e/packages/%40react-native-windows/tester/src/js/examples/Pressable/PressableExample.windows.js#L399

Reply:
If we align with `INativeKeyboardEvent` then the `key` is already 'correct' and someone just needs to change the Windows code